src/Bin/CodeGen/CodeGen.cpp

```diff
@@ -14,6 +14,7 @@
 #include "Engine/Tables/BuildingTable.h"
 #include "Engine/Events/EventMap.h"
 #include "Engine/Random/Random.h"
+#include "Engine/Objects/DecorationEnums.h"
```

Comment:
not sure that it is a right place for DecorationEnums.h"

Reply:
I might change some of this code later. Good enough for now.